uint32 id
float32 x
float32 y
float32 w
float32 x_vel
float32 y_vel
float32 w_vel
